warehouse/query-core/src/test/resources/datawave/query/QueryLogicFactory.xml

```diff
@@ -278,4 +278,7 @@
     
     <bean id="easyRoleManager" class="datawave.webservice.query.logic.EasyRoleManager" />
     
+    <util:list id="RemoteUserServices" value-type="datawave.webservice.common.remote.RemoteUserOperations">
```

Comment:
Used to configure the UserOperationsBean with any additional remote user operations